basicsettings.ml

```diff
@@ -256,8 +256,26 @@ module Unify = struct
   let infer_recursive_types = Settings.add_string("infer_recursive_types", "guarded", `User)
 end
 
-(* Should we use the extra standard library definitions? *)
-let use_stdlib = Settings.add_bool ("use_stdlib", true, `User)
+(* Standard Library stuff *)
+module StdLib = struct
+  (* Should we use the extra standard library definitions? *)
+  let use_stdlib = Settings.add_bool ("use_stdlib", true, `User)
 
-(* Standard library path *)
-let stdlib_path = Settings.add_string ("stdlib_path", "", `User)
+  (* Standard library path *)
+  let stdlib_path = Settings.add_string ("stdlib_path", "", `User)
+end
+
+module Readline = struct
+  (* Path for readline history file *)
+  let readline_history_setting = Settings.add_string("readline_history_path", "", `User)
```

Comment:
This option may be unnecessary. It seems to be of little value. On the other hand, I don't mind it either.